Lib/turtle.py

```diff
@@ -1939,31 +1939,40 @@ def circle(self, radius, extent = None, steps = None):
         """ Draw a circle with given radius.
 
         Arguments:
-        radius -- a number
-        extent (optional) -- a number
+        radius -- a number (distance)
+        extent (optional) -- a number (angle, in degrees)
         steps (optional) -- an integer
+        
+        With one argument, draw a circle with the given radius. The
+        center is radius units to the left of the turtle. If radius is
+        negative, the center is to the right.
+
+        If extent is given, do not draw the whole circle, but only an
+        arc of the circle extent degrees wide starting from the current
+        position. If extent is negative, draw the arc while moving
+        backwards around the circle from the current position. In either
+        case the final heading of the turtle will be extent degrees
+        different from the original heading.
+
+        The circle or arc drawn is not a true geometric curve (impossible
+        on a computer screen composed of pixels), but rather is composed
+        of many very short straight steps. The number of steps to use is
+        calculated automatically to give the appearance of a true curve.
```

Comment:
In order to keep the docstring more concise, I would merge this paragraph with the next and remove the part about why the circle is created by steps.  Saying that it's done in steps, that the number is calculated automatically if not specified, and that this can be used to draw polygons it's enough IMHO.